tango_simlib/tango_sim_generator.py

```diff
@@ -211,17 +211,33 @@ def _reset_to_default_state(self):
             """Reset the model's quantities' adjustable attributes to their default
             values.
             """
+            type_map = {
+                'DevString': "",
+                'DevFloat': 0.0,
+                'DevDouble': 0.0,
+                'DevBoolean': False,
+                'DevEnum': 0,
+                'DevLong': 0,
+                'DevULong': 0,
+                'DevVoid': None}
```

Comment:
Nice, but we have a similar constant already `INITIAL_CONSTANT_VALUE_TYPES`, so rather add the missing types to that one and reuse it.
https://github.com/ska-sa/tango-simlib/blob/47799189bcce985788bfce356150be82298e978e/tango_simlib/model.py#L36